plugins/modules/sonic_radius_server.py

```diff
@@ -89,7 +89,13 @@
             suboptions:
               name:
                 description:
-                  - Specifies the name of the radius server host.
+                  - Specifies the IP address or name of the radius server host.
+                  - A host name may start with 1 to 63 characters including
+                  - any of the following allowed characters, but cannot
+                  - start or end with a hyphen.
+                  - Allowed characters are a through z, A through Z, 0 through 9 and hyphen.
+                  - The host name must end with a dot (.) character followed by two to six
+                  - additional lower or upper case alpabetic characters. (a through z, A through Z)
```

Comment:
Please provide a reason for this suggested change.

I added the description of format requirements because I think that it will be helpful to users. (I found it difficult to determine the revised rules for host names and would like to save our users this trouble. Unlike me, they don't have access to the SONiC source code.) I am willing to consider the proposed line deletions if a good reason is provided for doing them.

Reply:
The following reason was provided for making this change, and I have proceeded with the change:

```
regarding the hostname description in the PR, the details of a valid hostname could be removed, since the format supported by SONiC is a standard format.
```
The same standard format is already used in other resource modules and users are likely to already be aware of it.